### DISCUSSION TITLE
can't access the huggingface demo

### DISCUSSION THREAD

**Discussion by sanbuphy** [PixArt-alpha/PixArt-alpha#2] [Q&A] [answered]
can't access the huggingface demo

**Comment by Njasa2k** [comment]
Same

**Comment by lawrence-cj** [comment]
Still working on the project. Plz stay tuned.

**Comment by yujincheng08** [comment, marked as answer]
online now.

https://huggingface.co/spaces/PixArt-alpha/PixArt-alpha
